widget/entry.go

```diff
@@ -507,11 +524,14 @@ func (e *Entry) TappedSecondary(pe *fyne.PointEvent) {
 
 	c := fyne.CurrentApp().Driver().CanvasForObject(e)
 
-	item := fyne.NewMenuItem("Paste", func() {
+	pasteItem := fyne.NewMenuItem("Paste", func() {
 		clipboard := fyne.CurrentApp().Driver().AllWindows()[0].Clipboard()
 		e.pasteFromClipboard(clipboard)
 	})
-	e.popUp = NewPopUpMenu(fyne.NewMenu("", item), c)
+	selectAllItem := fyne.NewMenuItem("Select all", func() {
```

Comment:
instead of creating an anonymous function, just pass a.selectAll as the function parameter